ResearchKit/1.4.1/ResearchKit.podspec

```diff
@@ -1,15 +0,0 @@
-Pod::Spec.new do |s|
```

Comment:
Yeah, this is intentional. It seems like an unused duplicate of `ResearchKit.podspec` on the root folder.